Improved the pritive dialogs

Micheus opened the pull request:
- Removed the Image Plane thickness;
- Added dialog option to Octahedron and Octotoad objects;
- Added new options: Rotate, Move and Put On Ground;
- It was standardized the dialogs by using label_column always as possible.

dgud commented on src/wings_edge_cmd.erl:
```diff
@@ -180,6 +189,12 @@ connect(Es0, #we{id=Id}=We0, Acc) ->
+connect_slide(St1) ->
+    {St0,Sel} = wings_sel:mapfold(fun connect/3, [], St1),
+    St=wings_sel:set(Sel, St0),
+    wings_wm:psend(geom, {save_state,St}),
```
You should not use psend() here,  psend should only be used to send messages between processes.
Also that will cause the undo order to wrong. Maybe remove that line?

dgud commented:
Hmm you should really make separate PR for this issues since they do different things, and it becomes very confusing when I comment on different commits.

Micheus commented:
Dan, I already separated the commits into PR as you asked.
For the Image Plane I added a MMB option to create the 'no thickness' object since some users already asked for that option;
I also removed the psend command as you suggested for connect_slide. I'm not sure why I thought I would need it.
